Formula/librealsense@1.12.1.rb

```diff
@@ -0,0 +1,37 @@
+class LibrealsenseAT1121 < Formula
+  desc "Intel RealSense F200, SR300, R200, LR200 and ZR300 capture"
+  homepage "https://github.com/IntelRealSense/librealsense/tree/legacy"
+  url "https://github.com/IntelRealSense/librealsense/archive/v1.12.1.tar.gz"
+  sha256 "62fb4afac289ad7e25c81b6be584ee275f3d4d3742468dc7d80222ee2e4671bd"
+  head "https://github.com/IntelRealSense/librealsense.git", :branch => "legacy"
+
+  option "with-examples", "Install examples"
```

Comment:
don't need this

Reply:
The examples flag was present before the bump to the 2.x branch and can add (unnecessary) compile time if default on: baf5d33b2f32effc216eaae58f61cff18fdcd9bb

If you're suggesting the examples are not necessary, then I'd disagree. They are useful for debugging the devices.

Reply:
Either we include them always or exclude them always (I'll leave it up to you). I don't want options on a new versioned formula.

Reply:
Ok, sounds fine, I'll include them.